app/models/org.rb

```diff
@@ -141,7 +141,7 @@ class Org < ApplicationRecord
   before_validation :check_for_missing_logo_file
 
   # This gives all managed orgs api access whenever saved or updated.
-  before_save :ensure_api_access, if: ->(org) { org.managed? }
+  before_save :ensure_api_access, if: lambda(&:managed?)
```

Comment:
I get the same Rubocop "fix" recommendation. However, I think this change, as well as `after_update :reconcile_published, if: ->(template) { template.published? }` (within `app/models/template.rb`) are leading to errors.